interface/billing/ub04_form.php

```diff
@@ -0,0 +1,1404 @@
+#ub04id403{left:324px; top:1102px; width:110px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id404{left:436px; top:1102px; width:132px; height:14px; color: rgb(0,0,0); text-align:right;}
+#ub04id405{left:622px; top:1102px; width:22px; height:14px; color: rgb(0,0,0); text-align:left;    /*font:normal 10px 'Times New Roman', Times, serif;*/}
+#ub04id406{left:662px; top:1102px; width:113px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id407{left:798px; top:1102px; width:23px; height:14px; color: rgb(0,0,0); text-align:left;    /*font:normal 10px 'Times New Roman', Times, serif;*/}
+#ub04id408{left:821px; top:1102px; width:98px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id409{left:72px;  top:1102px; width:209px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id410{left:303px; top:1120px; width:20px; height:14px; color: rgb(0,0,0); text-align:center;}
+#ub04id411{left:324px; top:1120px; width:110px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id412{left:436px; top:1120px; width:132px; height:14px; color: rgb(0,0,0); text-align:right;}
+#ub04id413{left:594px; top:1120px; width:170px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id414{left:790px; top:1120px; width:129px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id415{left:19px;  top:1120px; width:263px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id416{left:303px; top:1137px; width:20px; height:14px; color: rgb(0,0,0); text-align:center;}
+#ub04id417{left:324px; top:1137px; width:110px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id418{left:436px; top:1137px; width:132px; height:14px; color: rgb(0,0,0); text-align:right;}
+#ub04id419{left:622px; top:1137px; width:22px; height:14px; color: rgb(0,0,0); text-align:left;    /*font:normal 10px 'Times New Roman', Times, serif;*/}
+#ub04id420{left:662px; top:1137px; width:113px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id421{left:821px; top:1137px; width:98px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id422{left:798px; top:1137px; width:23px; height:14px; color: rgb(0,0,0); text-align:left;    /*font:normal 10px 'Times New Roman', Times, serif;*/}
+#ub04id423{left:19px;  top:1137px; width:263px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id424{left:303px; top:1155px; width:20px; height:14px; color: rgb(0,0,0); text-align:center;}
+#ub04id425{left:324px; top:1155px; width:110px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id426{left:436px; top:1155px; width:132px; height:14px; color: rgb(0,0,0); text-align:right;}
+#ub04id427{left:594px; top:1155px; width:170px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id428{left:790px; top:1155px; width:129px; height:14px; color: rgb(0,0,0); text-align:left;}
+#ub04id429{left:19px;  top:1155px; width:263px; height:14px; color: rgb(0,0,0); text-align:left;}
```

Comment:
I am afraid to ask how much time it took for you to come up with all the px measurements :)